test/test_mercado_pago.rb

```diff
@@ -1,17 +1,25 @@
 # encoding: utf-8
 
+require File.expand_path('../../lib/mercadopago', __FILE__)
+
 require 'test/unit'
-require 'mercadopago'
+require 'vcr'
+
+#
+# Valid credentials to be used in the tests.
+#
+CREDENTIALS = {
+  client_id: '3962917649351233',
+  client_secret: 'rp7Ec38haoig7zWQXekXMiqA068eS376' }
```

Comment:
@schmierkov Please remove real credentials from the test. We shouldn't depend on these being valid to test our code. If we make sure that these are not real, we can be sure that we are not really connecting to Mercado Pago.